JesusFilm/core
# fix: conductor rtl

# Description

Conductor should navigate correctly on RTL and LTR journeys

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413280154)

# How should this PR be QA Tested?

Make an RTL journey (eg locale is Arabic) in journeys-admin (use vercel link)
OR
Request access to premade journeys (if you want to edit)
LTR https://admin-stage.nextstep.is/journeys/3148f59e-9d40-4a52-aa5c-46830349f08a
RTL https://admin-stage.nextstep.is/journeys/7f28e898-d168-46a1-8614-d14080f63212
OR
Use premade journey direct link
https://your-stage.nextstep.is/ltr-journey-demo
https://your-stage.nextstep.is/rtl-journey-demo-2

Open journeys with **journeys** vercel link  + slug of your own RTL or premade journeys: `https://journeys-git-22-08-zl-fix-conductor-rtl-jesusfilm.vercel.app/journeys/[slug]`

For LTR
- [x] it should hide nav arrow on the **left** if on first card of journey
- [x] it should be able to click on nav arrow on the **right** by default
- [x] it should disable nav arrow on the **left** by default
- [x] it should disable nav arrow on the **right** when next card is locked
- [x] it should hide nav arrow on the **right** if on last card of journey
- [x] it should left align the copyright notice

For RTL
- [x] it should hide nav arrow on the **right** if on first card of journey
- [x] it should be able to click on nav arrow on the **left** by default
- [x] it should disable nav arrow on the **right** by default
- [x] it should disable nav arrow on the **left** when next card is locked
- [x] it should hide nav arrow on the **left** if on last card of journey
- [x] it should right align the copyright notice

These should be correct in the journey and in Conductor stories in storybook

# Checklist
